### ISSUE TITLE
Which sensitivity LSB range was used?

### ISSUE THREAD

**Issue by papamauro** [mmalekzadeh/motion-sense#1]
Hello,
I would like to know which is the sensitivity range set in the accelerometer used to get this data. This information would allow to convert data from Gs units to other measure units.  After a small research i found Iphone 6s accelerometer datasheet. The Iphone 6s (just like the Iphone 6) [has two different chips](https://www.chipworks.com/about-chipworks/overview/blog/comparing-invensense-and-bosch-accelerometers-found-iphone-6), but probably [this](http://www.invensense.com/wp-content/uploads/2015/02/MPU-6500-Datasheet2.pdf) is the one used in the dataset. It turned out this chip has four different sensitivity levels to choose from while getting data:

±2g: 16384 LSB/g
±4g: 8192 LSB/g
±8g: 4096 LSB/g
±16g: 2048 LSB/g

Many thanks!

**Comment by mmalekzadeh** [comment]
That’s a good question.

In this study, we used [CrowdSense](https://itunes.apple.com/gb/app/crowdsense/id930853606?mt=8) app which works on top of [SensingKit](https://www.sensingkit.org/) for collecting users' data.

The developer of SensingKit told me: in iOS, you cannot set the sensitivity of the chip. Unfortunately (even though the chip might support multiple sensitivities). So, The short answer is, we don’t really know.

However, we did a quick test where we collected some Accelerometer data from an iPhone SE and iPhone 6s, by shaking the phone as fast as we can just to get the min maximum from all axis.
The result was:

Max: 8.061462 (for SE), 8.200882 (for 6s)
Min: -7.995941 (for SE), -8.18721 (for 6s)

So, we think it is safe to say that it uses the option:
±8g

You might want to do the same with the device you used, but we believe the results would be the same.

BTW, the phone model which have been used for collecting the MotionSense dataset is Apple iPhone 6.
